docs/sql-ref-syntax-ddl-drop-function.md

```diff
@@ -59,6 +59,18 @@ SHOW USER FUNCTIONS;
   | default.test_avg  |
   +-------------------+
 
+-- Create Temporary function `temp_avg`
+CREATE TEMPORARY FUNCTION temp_avg as 'org.apache.hadoop.hive.ql.udf.generic.GenericUDAFAverage';
+
+-- List user functions
+SHOW USER FUNCTIONS;
+  +-------------------+
+  |     function      |
+  +-------------------+
+  | default.test_avg  |
+  | temp_avg          |
```

Comment:
Are the functions having the same name?

Reply:
yes, one with `permanent` and other with `Temporary`

Reply:
@gatorsmile The permanent functions are always shown as database qualified where as the temporary functions are not qualified.

Reply:
oops :), sorry I corrected it 

![image](https://user-images.githubusercontent.com/35216143/65397892-5118e480-ddd1-11e9-8d2a-87d9839116be.png)